client/me/purchases/list/site/index.jsx

```diff
@@ -12,8 +12,9 @@ import i18n from 'i18n-calypso';
 import PurchaseItem from '../item';
 import SectionHeader from 'components/section-header';
 
-const PurchasesSite = ( { isPlaceholder, name, purchases, slug } ) => {
+const PurchasesSite = ( { isPlaceholder, name, purchases, slug, url } ) => {
 	let items, label = name;
+	const siteUrl = url && url.replace( /http:\/\//g, '' );
```

Comment:
What if there is a https protocol version provided? Code checks if url exists, so I assume it can be undefined, should we have any fallback here? :)